src/test_utils.rs

```diff
@@ -0,0 +1,4 @@
+pub fn trace_init() {
```

Comment:
I'd rather throw this into a `support.rs` file that is `cfg(test)` gated.

Reply:
Yeah, that's a good idea, I'll update it in the next push.